lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -149,6 +149,12 @@ def test_todays_date(self):
         self.assertEqual(block.title, 'Today is Jan 02, 2015')
         self.assertNotIn('date-summary-date', block.render())
 
+    @freezegun.freeze_time('2015-01-02')
```

Comment:
I had to move this test away from the base `DateSummary`, since it's isn't possible to mutate a block's date now that it's declared as `@property`.